galaxy_importer/galaxy-importer.cfg

```diff
@@ -0,0 +1,4 @@
+[galaxy-importer]
+LOG_LEVEL_MAIN = INFO
+RUN_ANSIBLE_TEST = False
```

Comment:
@anuj-apdev

Reply:
We should try without having this as well.
Initially we were not able to pass the `ansible-test sanity`, hence we have put this to bypass sanity test while import. Now since we are passing the `ansible-tests sanity`, we can get it removed.